README.md

```diff
@@ -3,34 +3,49 @@
-
-**Redwood is an opinionated, full-stack, serverless-ready web application framework
-that will allow you to build and deploy with ease.**
-Imagine a React frontend, statically delivered by CDN, that talks via GraphQL to
-your backend running on AWS Lambdas (or containers) around the world, all deployable with just a
-`git push`—that's Redwood. By making a lot of decisions for you, Redwood lets
-you get to work on what makes your application special, instead of wasting
-cycles choosing and re-choosing various technologies and configurations. Plus,
-because Redwood is a proper framework, you benefit from continued performance
-and feature upgrades over time and with minimum effort.
-
-Redwood is the latest open source project initiated by Tom Preston-Werner, cofounder of GitHub (most popular code host on the planet), creator of Jekyll (one of the first and most popular static site generators), creator of Gravatar (the most popular avatar service on the planet), author of the Semantic Versioning specification (powers the Node packaging ecosystem), and inventor of TOML (an obvious, minimal configuration language used by many projects).
-
-> **NOTICE:** RedwoodJS is very close to a stable version 1.0. In the last two years,
-> the project has matured significantly and is already used in production by a number
-> of startups. We intend to have a 1.0 release candidate before the end of 2021 and
-> to release a truly production-ready 1.0 in early 2022.
-
-> **TUTORIAL:** The best way to get to know Redwood is by going through the extensive
-> [Redwood Tutorial](https://redwoodjs.com/tutorial). Have fun!
-
-> **QUICK START:** You can install and run a full-stack Redwood application on your
-> machine with only a couple commands. Check out the [Quick Start](https://redwoodjs.com/docs/quick-start)
-> guide to get started.
-
-**EXAMPLES:** If you'd like to see some examples of what a Redwood application looks
-like, take a look at the following projects:
+_by Tom Preston-Werner, Peter Pistorius, Rob Cameron, David Price, and more than
+250 amazing contributors (see end of file for a full list)._
+
+**Redwood is an opinionated, full-stack, JavaScript/TypeScript web application
+framework designed to keep you moving fast as your app grows from side project
+to startup.**
+
+At the highest level, a Redwood app is a React frontend that talks to a custom
+GraphQL API. The API uses Prisma to operate on a database. Out of the box you
+get tightly integrated testing with Jest, logging with Pino, and a UI component
+catalog with Storybook. Setting up authentication (like Auth0) or CSS frameworks
+(like Tailwind CSS) are a single command line invocation away. And to top it
+off, Redwood's architecture allows you to deploy to either serverless providers
+(e.g. Netlify, Vercel) or traditional server and container providers (e.g. AWS,
+Render) with nearly no code changes between the two!
```

Comment:
maybe something like "... nearly no code changes when migrating between the two!"

Not necessarily sold on that either.